notebooks/249-oneformer-segmentation/249-oneformer-segmentation.ipynb

```diff
@@ -0,0 +1,409 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #45.            examples=[["sample.jpg", "semantic", "CPU"]], inputs=[inp_img, inp_task, inp_device]
</pre><p>I'm not sure that it is good idea to make device is part of examples (in some cases, user can not have specified device), please stay it out of example</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1049/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Done